src/main.cpp

```diff
@@ -192,6 +195,7 @@ int main(int argc, char *argv[])
   option long_options[] = {
     option{ "help", no_argument, nullptr, 'h' },
     option{ "version", no_argument, nullptr, 'V' },
+    option{ "usdt-file-activation", no_argument, nullptr, '$' },
```

Comment:
`$` seems like a weird flag, or it is a special thing in getopt long?

Reply:
It's only used for the `switch/case`. No short option for this flag.